_pages/members.md

```diff
@@ -383,6 +383,15 @@ order: 1
             Yosuke Higuchi
         </div>
     </div>
+    <div class="col-sm mt-1 mt-md-1">
+        <div class="square">
+        <a href="https://sites.google.com/view/yoshiki-masuyama/">
+            <img class="img-fluid rounded z-depth-1" src="{{ site.baseurl }}/assets/img/yoshiki.jpg">
+        </a></div>
+        <div class="caption">
+            Yoshiki Masuyama
+        </div>
+    </div>
     <div class="col-sm mt-3 mt-md-0">
     </div>
```

Comment:
```suggestion
```

Reply:
You can simple remove the lines to make it the same size as current layout

Reply:
Thank you for your suggestion.